test/src/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+if (CMAKE_BUILD_TYPE MATCHES Debug)
```

Comment:
this check for Debug no longer necessary since all of test/src is only build in Debug mode